app/assets/stylesheets/govuk_publishing_components/components/_inset-text.scss

```diff
@@ -6,3 +6,14 @@
     break-inside: avoid;
   }
 }
+
+.govuk-inset-text {
```

Comment:
Would there be any mileage in nesting this class inside `.gem-c-inset-text`? Just to make sure everything's water tight and we're limiting the scope of this override within a gem specific class.

Reply:
Good point, have done this now, thanks @JamesCGDS 👍